continuous-deployment/readme.md

```diff
@@ -1,7 +1,142 @@
 
-## Sections within Continuous Deployment
+Deploying software involves the following principles:
 
-* [Secrets Management](secrets-management/readme.md)
+1. Provision and manage the cloud environment runtime for your application (cloud resources, infrastructure, hardware, services, etc).
+
+1. Install the target application version across your cloud environments.
+
+1. Configure your application, including any required data.
+
+A continuous deployment pipeline is an automated manifestation of your process to streamline these very principles in a consistent and repeatable manner.
+
+## Goal
+
+* Follow industry best practices for delivering software changes to customers and developers.
+
+* Establish consistency for the guiding principles and best practices when assembling continuous delivery workflows.
+
+## General Guidance
+
+### Define a Release Strategy
+
+It's important to establish a common understanding between the technical lead and application stakeholder(s) around the release strategy / design  during the planning phase of a project. This common understanding includes the deployment and maintenance of the application throughout its SDLC.
+
+#### Release Strategy Principles
+
+*Continuous Delivery* by Jez Humble, David Farley cover the key considerations to follow when creating a release strategy:
+
+* _Parties in charge of deployments to each environment, as well as in charge of the release._
+
+* _An asset and configuration management strategy._
+
+* _An enumeration of the environments available for acceptance, capacity, integration, and user acceptance testing, and the process by which builds will be moved through these environments._
+
+* _A description of the processes to be followed for deployment into testing and production environments, such as change requests to be opened and approvals that need to be granted._
+
+* _A discussion of the method by which the application’s deploy-time and runtime configuration will be managed, and how this relates to the automated deployment process._
+
+* _Description of the integration with any external systems. At what stage and how are they tested as part of a release? How does the technical operator communicate with the provider in the event of a problem?_
+
+* _A disaster recovery plan so that the application’s state can be recovered following a disaster. Which steps will need to be in place to restart or redeploy the application should it fail._
+
+* _Production sizing and capacity planning: How much data will your live application create? How many log files or databases will you need? How much bandwidth and disk space will you need? What latency are clients expecting?_
+
+* _How the initial deployment to production works._
+
+* _How fixing defects and applying patches to the production environment will be handled._
+
+* How upgrades to the production environment will be handled, including data migration. How will upgrades be carried out to the application without destroying its state.
+
```

Comment:
I believe the section in the existing playbook which describes different environments would be helpful here, before `Environment Promotion` is described.